src/hooks/cart.js

```diff
@@ -13,8 +13,13 @@ export default class extends BaseHooks {
     }
 
     itemAdd() {
-        this.$body.on('submit', '[data-cart-item-add]', (event) => {
-            this.emit('cart-item-add', event, event.target);
-        });
+        const elements = document.querySelectorAll('[data-cart-item-add]');
+        if (elements.length) {
+            for (let i = 0; i < elements.length; i++) {
+                elements[i].addEventListener('submit', (event) => {
```

Comment:
I'd prefer to omit the parens for arrow functions with only one parameter. Same for similar instances below.

Reply:
fixed.